dace/transformation/transformation.py

```diff
@@ -418,14 +557,15 @@ def __init__(self,
             self.sdfg_id = sdfg_id
             self.state_id = state_id
 
-    def subgraph_view(self, sdfg: SDFG) -> SubgraphView:
+    def subgraph_view(self, sdfg: SDFG) -> gr.SubgraphView:
```

Comment:
SDFG without module name

Reply:
Same as above, I won't be able to fix all existing cases here.